mirage/utils/strings.js

```diff
@@ -0,0 +1,9 @@
+// Imported from the `@ember/string` package to avoid build issues...
```

Comment:
I mean, if this is all you need, then it's probably more valuable to remove the dep

Reply:
that's all we need in the app itself, but it looks like e.g. ember-tooltips needs more

Reply:
looks like ember-tooltips could also do with a v2 conversion :thinking: